dev/email-field.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+
+    <script type="module">
+      import '@vaadin/email-field';
+    </script>
+  </head>
+
+  <body>
+    <vaadin-email-field label="Required" required></vaadin-email-field>
+
+    <div>Is valid? <span id="validity"></span></div>
```

Comment:
This manual validity indicator feels unnecessary since the component already has a built-in indicator for it. Would remove all but the component.

Reply:
Done.